s3repo/view.py

```diff
@@ -27,42 +28,43 @@ def _readable_size(size):
                 return readable_size
             size /= 1024.0
 
-        return "%3.1f B" % (size)
+        return "%3.1f B" % size
```

Comment:
Not related with the commit.
What's the code style rule being violated here?

Reply:
Remove redundant parentheses